src/components/navigation/navigation-categories.ts

```diff
@@ -145,7 +145,7 @@ export const navigationCategories = [
           },
           {
             title: 'Founders',
-            url: '/horizon',
```

Comment:
please add a permanent redirect into next.config.js that sends /horizon to /founders. This will prevent any existing /horizon references (i.e. google search results) from breaking.